src/Core/Configure.php

```diff
@@ -421,14 +421,10 @@ public static function version(): string
         }
 
         $path = dirname(dirname(__DIR__)) . DIRECTORY_SEPARATOR . 'config/config.php';
-        if (file_exists($path)) {
```

Comment:
I don't see why this existence check was necessary. The `config/config.php` file is always going to exist in the cakephp package.

Reply:
Wasnt that because of phinx or sth?

Reply:
Came from https://github.com/cakephp/cakephp/pull/13942

Reply:
Hmm.. Right, it could be because of split packages.

Reply:
I reinstated the check and just changed the function.